tools/generateMetadata.ps1

```diff
@@ -6,6 +6,7 @@ Write-Host "GitHub user:" $Args[0]
 Write-Host "Branch:     " $Args[1]
 Try
 {
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12

```

Comment:
@shahabhijeet 
Checked a couple of blogs, and looks like this is good. Powershell `Invoke-RestMethod` seems to default to the tls 1.0 version which fails with a number of rest calls
An example [here](https://stackoverflow.com/questions/41618766/powershell-invoke-webrequest-fails-with-ssl-tls-secure-channel) 
I couldn't get any specific information regarding whether github also needs later versions of tls but I'm inclined to believe this is a useful change

Reply:
If you're running agains PS 6.0 or greater, you could also do "Invoke-RestMethod -SslProtocol Tls12" for a slightly cleaner solution, but I don't think the flag exists in earlier versions. Some info on github specifically: https://github.com/blog/1734-improving-our-ssl-setup and scan of their site: https://www.ssllabs.com/ssltest/analyze.html?d=github.com&s=192.30.255.112 
![image](https://user-images.githubusercontent.com/6800857/36755694-d2d18a8c-1bc1-11e8-97ce-01da931118b0.png)

Reply:
Ahh I missed that!